### DISCUSSION TITLE
Requesting NAIP Multispectral Data from USGS

### DISCUSSION THREAD

**Discussion by annieM459** [earthlab-education/Earth-Analytics-2023-01-Intro#553] [Q&A06 Vector Data] [answered]
Hi all! 

I am filling out the form for requesting the mutlispectral data and realizing there are a lot of holes in what I should be filling out. I am not sure what to put for these two categories. Any recomendations?

<img width="824" alt="Screenshot 2023-10-30 at 11 15 49 AM" src="https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/140002702/27e451d1-3146-461c-ac16-3017d995cf9d">

Thank you!

Annie

**Comment by jsmi374** [comment, marked as answer]
Hey Annie, 

You might look into the other discussion posting from last week for information on this titled "WEEK 8 Accessing Data Issue #550". I submitted a request last week for access, but still have not received approval. It will likely take a few days for you to gain access. I prepopulated all of my codespace cells with the code from the technical demo video, so that I will be ready when I do get access. 

Best Wishes, 

James

**Comment by eculler** [comment]
These questions are more for their record keeping than anything else, I think.

You can put NAIP as the product you want, and 500 scenes per year should more than cover it. We're very small potatoes on this, so it should be fine to round up.
